src/txn/txn_ckpt.c

```diff
@@ -1178,6 +1219,10 @@ __txn_checkpoint(WT_SESSION_IMPL *session, const char *cfg[])
     WT_WITH_SCHEMA_LOCK(session, ret = __checkpoint_prepare(session, &tracking, cfg));
     WT_ERR(ret);
 
+    /* Log the final checkpoint prepare progress message if needed. */
+    if (conn->ckpt_progress_msg_count > 0)
```

Comment:
If the checkpoint is quick enough, this means we won't print this as `ckpt_progress_msg_count` should be `0`. Is that expected?

Reply:
Yes, we do not want to log it for fast checkpoints.